### PR TITLE
Test: WorkFlow-service ContinuationService

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import org.mockito.Mockito;
+import org.mockito.internal.matchers.Matches;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (anludke) [inline workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java:29]: @eloycoto please use explicit import for each class

**Review by anludke — CHANGES_REQUESTED** [review]

File: workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+	private WorkFlowRepository workFlowRepository;
+
+	private WorkFlowTaskRepository workFlowTaskRepository;
+
+	private WorkFlowServiceImpl workFlowService;
+
+	private WorkFlowContinuationService service;
+
+	@Test
```
Comment (pkliczewski) [inline workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java:47]: Why do we need this annotation here?

**Review by pkliczewski — COMMENTED** [review]
Overall looks good. Only one small comment.

File: workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java (hunk 2 above)
Comment (eloycoto) [reply workflow-service/src/test/java/com/redhat/parodos/workflow/execution/continuation/WorkFlowContinuationServiceTest.java]: intelj magic :sweat_smile:

**Review by anludke — APPROVED** [review]

**Pull request merged**